modules/javafx.web/src/main/java/com/sun/webkit/graphics/WCRenderQueue.java

```diff
@@ -171,6 +171,15 @@ public boolean isOpaque() {
         return opaque;
     }
 
+    /*is called from native*/
+    private synchronized boolean strokeContains(WCPath path, float x, float y) {
+        if (gc == null) {
+            return false;
+        }
+        decode();
```

Comment:
Is calling `decode` from a thread other than the PrismRenderer thread a safe thing to do? I realize that you have synchronized this, which is necessary, but is it sufficient?